Makefile

```diff
@@ -5,7 +5,7 @@ openapi-generator-cli:=java -jar $(openapi-generator-jar)
 
 generator:=java
 library:=okhttp-gson
-services:=payments checkout
+services:=balanceplatform payments checkout
```

Comment:
Should we add the service ?

Reply:
What do you mean exactly? This is the specific Configuration API endpoints from the Balance Platform, which for some reason in the specs are also called balanceplatform. Or do you mean something else?